web/src/utils/panel.tsx

```diff
@@ -0,0 +1,3 @@
+export const defaultSize = '500px';
+export const minSize = '250px';
+export const maxSize = '750px';
```

Comment:
These values are comfortable on my resolutions. Feel free if you feel we should move to `%`
https://www.patternfly.org/v4/components/drawer#drawerpanelcontent